src/controllers/construction.ts

```diff
@@ -64,11 +63,10 @@ const processPayloads = async (req: Request, res: Response, next: NextFunction)
                 && operation.metadata && operation.metadata.public_key && operation.metadata.public_key.hex_bytes
                 && operation.metadata.public_key.curve_type === CurveTypesEnum.ED25519
         }) as IOperation;
-
         if (!operationForTransfer) {
             throw new ErrorResponse(ErrorCodes.BadOperationForConstruction, `Cant find operation to build tx, check type, symbol, decimals or value`);
         }
-
+        //Will not work with the use of the rosetta cli as there is no metadata being passed in by lto.ros
```

Comment:
What is the consequence of this? We need that in order to function right? It is available in the our own test from what I see in this PR correct?

Reply:
Is a proper error thrown when the metadata on which it relies is not present? If not it makes sense to do that.

Reply:
Okay, then I do suggest to include it in the README. So that a user of the API knows what value to pass into the endpoint

Reply:
Added a postman collection with examples for every call that can be made to the API

Reply:
https://www.getpostman.com/collections/3c2028a379f2297914af